platform/marvell-armhf/sonic-platform-et6448m/inband_mgmt.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+#inband_mgmt
+
+inband_mgmt(){
+
+# The intent of this sequence is to ensure 12c bus enumeration order by
+# controlling the order in which the various i2c device drivers are
+# loaded. Hence the drivers are unloaded and then reloaded in the prescribed
+# order.
+# NOTE: In the nokia platform the following sequence is performed by the Nokia
+# platform service init script and thus should not be performed here
+ grep ^onie_platform /host/machine.conf 2>/dev/null | grep nokia >/dev/null
+ if [ $? != 0 ]; then
+     rmmod i2c-dev
+     rmmod i2c_mux_gpio
+     rmmod i2c_mv64xxx
+     modprobe i2c_mv64xxx
+     modprobe i2c-dev
+     modprobe i2c_mux_gpio
+     sleep 60
```

Comment:
such sleep concerns me. why do we need so long wait time.

Reply:
Here, the logic is to start a monitoring loop for the creation of in-band management port once data ports are initialized through sai. 
So sleep has been introduced to start later after syncd docker is initialized.